bolt-modules/boltlib/lib/puppet/functions/add_to_group.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'bolt/error'
+
+# Adds a target to specified inventory group.
+Puppet::Functions.create_function(:add_to_group) do
+  # @param target A Target object from implicitly passed by calling method.
+  # @param The name of the group to add target to.
+  # @example Add new target to group.
+  #   Target.new('foo@example.com', 'password' => 'secret').add_to_group('group1')
+  # @example Add new target to group by name.
+  #   add_to_group('bolt:bolt@web.com', 'group1')
+  dispatch :add_to_group do
+    param 'Boltlib::TargetSpec', :target
```

Comment:
This could legally be multiple targets. And I think we should allow that. So the name should be `targets`, use `get_targets` to convert it to an array of targets, and pass that to Inventory.

Reply:
Can you change the param symbol to `targets`?